test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,22 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() {
```

Comment:
This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.